### PR TITLE
increased activeOpacity and updated default to primary

### PR DESCRIPTION
**Pull request by MD-REHMAN**

### CODE REVIEW

File: src/components/primitives/Checkbox/types.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export type ICheckboxProps = IBoxProps & {
   id?: string;
   name?: string;
   value: ICheckboxValue;
-  colorScheme?: string | 'default';
+  colorScheme?: string | 'primary.';
```
Comment (Ajamuar) [inline src/components/primitives/Checkbox/types.tsx]: `primary.`?

File: example/storybook/stories/components/primitives/Radio/playground.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default function () {
           <Text mx={2}>Are you Awesome?</Text>
         </Radio>
         <Radio
-          colorScheme={text('colorScheme', 'default')}
+          colorScheme={text('colorScheme', 'primary.')}
```
Comment (Ajamuar) [inline example/storybook/stories/components/primitives/Radio/playground.tsx]: `primary.`

File: src/components/primitives/Radio/types.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ export type IRadioGroupOnChangeHandler = (value: IRadioValue) => any;
 
 export type IRadioProps = IBoxProps & {
   value: IRadioValue;
-  colorScheme?: string | 'default';
+  colorScheme?: string | 'primary.';
```
Comment (Ajamuar) [inline src/components/primitives/Radio/types.tsx]: `primary.`?

**Review by Ajamuar — CHANGES_REQUESTED** [review]

**Comment by MD-REHMAN** [comment]
No longer required.

**Pull request closed without merging**